Paperwork Items Fixed!

darkincarnate opened the pull request:
## About the Pull Request

- Added extra folders, paper shredders and filing cabinets to most, if not all offices.

- Nuke envelope in SD inner-office removed, the secure briefcase already has one inside it.

- EC Liason Cigarette Machine and Snack machine gone, they didn't need them and they get a paper shredder and chest drawer instead!

## Why It's Good For The Game

This is good for the game as there is real storage and basic paperwork items for jobs that pretty much require paperwork to function.

## Changelog

:cl:
add: Added paper shredder and wall-filing cabinets to all offices if they didn't have them already.
add: Added more things
del: Removed one of the nuke envelopes in SD office, they don't need two.
/:cl:

fixes https://github.com/Foundation-19/Foundation-19/issues/1128

darkincarnate committed:
Paperwork Items Fixed!

- Added extra folders, paper shredders and filing cabinets to most, if not all offices.

- Nuke envelope in SD inner-office removed, the secure briefcase already has one inside it.

darkincarnate commented:
https://github.com/Foundation-19/Foundation-19/issues/1128#issue-1680156363

aidan-bowman commented:
Edit in "fixes #1128" somewhere in the PR description so that the issue is properly linked.

darkincarnate committed:
GOI Office Revamp

- Made the GOI Representative's officer bigger, gave them filing cabinets, gave them a safe and put some things in their locker (money case and radio).

darkincarnate committed:
Revert "GOI Office Revamp"

This reverts commit 1630367d671850059c1bc3898aed4c3c085d3f74.

darkincarnate committed:
Merge branch 'dev' into random-map

Tsurupeta committed:
Merge branch 'dev' into random-map
